### PR TITLE
Added a method to get count of users

### PR DESCRIPTION
**Pull request by Pritoj**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
FEATURE

* **What is the current behavior?** (You can also link to an open issue here)
#26 

* **What is the new behavior (if this is a feature change)?**
This basically helps you get the count of the entries. Helpful for pagination.

### CODE REVIEW

**Comment by Pritoj** [comment]
Hey guys, can you help me resolve this? I'm unable to figure out where this is failing.

Thanks for the help

**Comment by lucaperret** [comment]
Hello @Pritoj,
Could you rebase your branch on master ?
Strapi's team has been rollback the pluralized content types.

**Comment by Pritoj** [comment]
@lucaperret Done good sir.

File: src/lib/sdk.ts [hunk 1]
```diff
@@ -241,6 +241,20 @@ export default class Strapi {
     });
   }
 
+  /**
+   * Get the total count of entries with the provided criteria
+   * @param contentTypePluralized
+   * @param params Filter and order queries.
+   */
+  public getEntryCount(
+    contentTypePluralized: string,
```
Comment (lucaperret) [inline src/lib/sdk.ts]: Could you change all `contentTypePluralized` occurrence to `contentType`.

Comment (Pritoj) [reply src/lib/sdk.ts]: Yes sir. Fixing now.

Comment (Pritoj) [reply src/lib/sdk.ts]: It is done.

**Comment by lucaperret** [comment]
Thanks a lot @Pritoj !

**Pull request merged**